aries-site/src/layouts/main/Layout.js

```diff
@@ -143,6 +176,51 @@ export const Layout = ({
                 </Box>
               </Main>
             </MainContentWrapper>
+            <FeedbackButton
+              margin={{ vertical: 'large', horizontal: 'medium' }}
+              elevation="large"
+              onClick={onOpen}
+              color="purple!"
+              label="Feedback"
+              primary
+              a11yTitle={`This button launches a modal to give feedback.`}
+            />
+            <Feedback
+              onEsc={onClose}
+              onClose={onClose}
+              show={open}
+              messages={{
+                submit: 'Submit feedback',
+                cancel: 'No thanks',
+                successful: 'Thank you!',
+              }}
+              isSuccessful={isSucessful}
+              modal
+              title="We’d love your feedback"
+              onChange={nextValue => setValue(nextValue)}
+              onReset={() => setValue(defaultFeedback)}
+              onSubmit={onSubmit}
+              layerProps={{
+                position: !['xsmall', 'small'].includes(breakpoint)
+                  ? 'bottom-right'
+                  : 'center',
+                margin: { vertical: 'xlarge', horizontal: 'medium' },
+              }}
+            >
+              <Question
+                label="Was this page helpful"
+                kind="thumbs"
+                name="like-rating"
+              />
+              <Question
+                name="text-area"
+                kind="textArea"
+                label="Comments"
```

Comment:
```suggestion
                label="Any additional comments?"
```
Feels like we should be phrasing this as a question if possible to be more personable?